src/storage.h

```diff
@@ -21,13 +21,19 @@ enum ColumnFamilyID{
   kColumnFamilyIDMetadata,
   kColumnFamilyIDZSetScore,
   kColumnFamilyIDPubSub,
+  kColumnFamilyIDPropagate,
 };
 
 namespace Engine {
 extern const char *kPubSubColumnFamilyName;
 extern const char *kZSetScoreColumnFamilyName;
 extern const char *kMetadataColumnFamilyName;
 extern const char *kSubkeyColumnFamilyName;
+extern const char *kPropagateColumnFamilyName;
+
+extern const char *kPropagateTypeLua;
```

Comment:
```suggestion
extern const char *kPropagateLuaCommand;
```
this is a key name, i think `type` word is not clear.

`key` | `value`
lua -> script load
lua -> script flush

Reply:
agree